src/modules/subscription/domain/services/subscription.service.spec.ts

```diff
@@ -24,8 +25,8 @@ describe('SubscriptionService', () => {
       ],
     }).compile();
 
-    service = module.get<SubscriptionService>(SubscriptionService);
-    repository = module.get<ISubscriptionRepository>(
+    service = module.get<SubscriptionService>(SubscriptionServiceImpl);
+    repository = module.get<SubscriptionRepository>(
```

Comment:
I am just curious:
I see that the interfaces lost their prefix "I": ISubscriptionRepository -> SubscriptionRepository. 
However, the implementations received their postfix "Impl": SubscriptionService -> SubscriptionServiceImpl.

What is the logic behind it?)

Reply:
This is also related to Robert Martin's staffs))) In Clean Code, Uncle Bob described that it's better to use 'Impl' in implementations, instead of using the prefix 'I' in interfaces. I fixed it because all other interfaces don't use this prefix 'I', so I made it all consistent with the other files.